Ginger/GingerCore/SourceControl/SVNSourceControl.cs

```diff
@@ -501,15 +506,11 @@ public override bool ResolveConflicts(string Path, eResolveConflictsSide side, r
             if (client == null) Init();
             try
             {
-                if (System.IO.Path.GetExtension(Path) != string.Empty && !System.IO.File.Exists(System.IO.Path.GetFullPath(Path.Replace(".xml", ".ignore"))))
-                {
-                    System.IO.File.Copy(Path, (System.IO.Path.GetFullPath(Path.Replace(".xml", ".ignore"))));
-                }
-                CleanUp(Path);
+                CleanUp(System.IO.Path.GetDirectoryName(Path));
```

Comment:
check if cleanup for specific path works

Reply:
For SVN we have to give working directory path to Cleanup function. If we pass file path getting "Not a working directory" exception.